tasks/e2e-simple.sh

```diff
@@ -127,7 +127,7 @@ yarn build
 exists build/*.html
 exists build/static/js/*.js
 exists build/static/css/*.css
-exists build/static/media/*.svg
+# exists build/static/media/*.svg
```

Comment:
Given that our template changes are now in a separate repo - these changes are probably redundant now. You can try removing the comments.

Reply:
Will review this with setting up CI